en/running-desktop-app.md

```diff
@@ -0,0 +1,15 @@
+
+## Running the Desktop app
+Stay connected to your apps on any device Everyone needs instant access to their work no matter where they are – we’ve got you covered! Mobile, desktop, or web, App Builder is designed, so you can empower your team to reach its full potential from any device, even on the go.
```

Comment:
Stay connected to your apps on any device. Everyone needs
Are you missing a dot before Everyone?